app/src/main/java/com/yeonae/chamelezone/view/login/LoginFragment.kt

```diff
@@ -45,19 +54,42 @@ class LoginFragment : Fragment() {
     }
 
     private fun loginCheck(email: String, password: String) {
-        if (email.isEmpty()) {
-            Toast.makeText(context!!.applicationContext, "아이디를 입력해주세요!", Toast.LENGTH_SHORT).show()
-        } else if (password.isEmpty()) {
-            Toast.makeText(context!!.applicationContext, "비밀번호를 입력해주세요!", Toast.LENGTH_SHORT).show()
-        } else {
-            if (email == testEmail) {
-                if (password == testPassword) {
-                    Toast.makeText(context!!.applicationContext, "로그인 성공", Toast.LENGTH_LONG).show()
-                } else {
-                    showDialog()
+        when {
+            email.isEmpty() -> Toast.makeText(requireContext(), "아이디를 입력해주세요!", Toast.LENGTH_SHORT).show()
+            password.isEmpty() -> Toast.makeText(context!!.applicationContext, "비밀번호를 입력해주세요!", Toast.LENGTH_SHORT).show()
```

Comment:
```suggestion
            password.isEmpty() -> Toast.makeText(requireContext(), "비밀번호를 입력해주세요!", Toast.LENGTH_SHORT).show()
```
위와 일치시켜 주는것이 좋을 것 같아요

Reply:
수정했습니당